### DISCUSSION TITLE
problem installing Github Desktop for Mac<title>

### DISCUSSION THREAD

**Discussion by joegerm** [community/community#44802] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Cannot complete the installation of Github Desktop on a Mac running Monterey - Error invoking remote method 'move-to-applications-folder': Error: Failed to copy to applications directory even with authorization

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @joegerm ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by jge162** [reply]
https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/overview/supported-operating-systems

here is GitHub desktop details on how to fix. @joegerm you can also click on known issues as well. this should answer your question hopefully...
